packages/react-components/src/Button/types.ts

```diff
@@ -9,7 +9,7 @@ export type Button$Callback = () => void | Promise<void>;
 export interface ButtonProps {
   children?: React.ReactNode;
   className?: string;
-  icon: IconName;
+  icon?: IconName;
```

Comment:
Icons are required on all Buttons. We really don't want to do this, it was literally months of work to make sure we have icons everywhere enabling us to switch this on.